src/client/lib/statefulUrl.js

```diff
@@ -0,0 +1,48 @@
+const { compressToEncodedURIComponent, decompressFromEncodedURIComponent } = require('lz-string');
```

Comment:
Why did you change this from `import { compressToEncodedURIComponent as encode, decompressFromEncodedURIComponent as decode } from 'lz-string';`? If we're on ES6, we should make use of it throughout the project

Reply:
Didn't know how to do it correctly... now I do 😀 - and I've changed it accordingly